src/components/movie/full_movie_details.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import { 
+  getFullMovieDetails, 
+  getMovieSuggestions
+} from 'actions';
+import { PageLoader } from 'components/loader';
+
+import MainMovieDetails from 'components/main_movie_info';
+import SimilarMovies from 'components/similar_movies';
+import Screenshots from 'components/screenshots';
+import MovieSubinfo from 'components/movie_subinfo';
+import MovieTrailerModal from 'components/movie_trailer_modal';
+import TorrentsModal from 'components/torrents_modal';
```

Comment:
This pattern of importing every component in a separate line can be tiresome.
To evade the repeating pattern maybe add an index.js file in root of the components folder and export all the components from that file. So later on you can import them like this:
import { MainMovieDetails, SimilarMovies, Screenshots, etc... } from 'components';